SDK/AppCenterData/Microsoft.AppCenter.Data.iOS/Data.cs

```diff
@@ -25,16 +25,12 @@ public partial class Data : AppCenterService
 
         static Data()
         {
-            _remoteOperationDelegate.DataDidCompleteRemoteOperation = (operation, documentMetadata, error) =>
+            _remoteOperationDelegate.OnDataDidCompleteRemoteOperationAction = (operation, documentMetadata, error) =>
             {
-                if (RemoteOperationCompleted == null)
-                {
-                    return;
-                }
-                RemoteOperationCompleted(typeof(Data), new RemoteOperationCompletedEventArgs
+                RemoteOperationCompleted?.Invoke(null, new RemoteOperationCompletedEventArgs
                 {
                     Operation = operation,
-                    DocumentMetadata = documentMetadata.ToDocumentMetadata(),
+                    DocumentMetadata = documentMetadata?.ToDocumentMetadata(),
```

Comment:
I was looking at Android code and it does not have the null check.
What scenario causes metadata to be null? Is that real?

Reply:
metadata won't be null, null check wouldn't hurt, is there a way to enforce non null arguments?

Reply:
if it's not null, we usually don't put `?.`, that's confusing.